<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeSelector</class>
 <widget class="QDialog" name="NodeSelector">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Folder Selection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#cbAlwaysUploadToLocation
{
    spacing: 10px;
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font/>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Select a MEGA folder:</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="tMegaFolders">
     <property name="autoExpandDelay">
      <number>-1</number>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>25</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAlwaysUploadToLocation">
     <property name="text">
      <string>Always upload to this destination</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bLayout" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>26</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bNewFolder">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>New folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bOk">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>label</tabstop>
  <tabstop>tMegaFolders</tabstop>
  <tabstop>cbAlwaysUploadToLocation</tabstop>
  <tabstop>bNewFolder</tabstop>
  <tabstop>bOk</tabstop>
  <tabstop>bCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>NodeSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
